Makefile

```
@@ -0,0 +1,59 @@
+.PHONY: build docker test clean prepare update
+
+#GOOS=linux
+
+GO=CGO_ENABLED=0 GO11MODULE=on go
+GOCGO=CGO_ENABLED=1 GO111MODULE=on go
```

Line 5 missing a 1 in GO11MODULE.

This variable can be removed in both lines, because Go 1.16 turns GO111MODULE=on by default. Go 1.17 ignores it alltogether: https://go.dev/blog/go116-module-changes

```suggestion
GO=CGO_ENABLED=0 go
GOCGO=CGO_ENABLED=1 go
```